website/docs/docs/collaborate/data-tile.md

```diff
@@ -120,12 +120,34 @@ Follow these steps to embed the data health tile in Tableau:
 3. Insert a **Web Page** object.
 4. Insert the URL and click **Ok**.
 
-    `https://metadata.cloud.getdbt.com/exposure-tile?uniqueId=exposure.snowflake_tpcds_sales_spoke.customer360_test&environmentType=production&environmentId=220370&token=<YOUR_METADATA_TOKEN>`
-    
+    ```html
+    https://metadata.ACCESS_URL/exposure-tile?uniqueId=exposure.EXPOSURE_NAME&environmentType=production&environmentId=220370&token=<YOUR_METADATA_TOKEN>
+    ```
+
     *Note, replace the placeholders with your actual values.*
 5. You should now see the data health tile embedded in your Tableau dashboard.
 
 </TabItem>
+
+<TabItem value="sigma" label="Sigma example">
+
+Follow these steps to embed the data health tile in Sigma:
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/sigma-example.jpg" width="90%" title="Embed data health tile in Sigma"/>
+
+1. Create a dashboard in Sigma and connect to your database to pull in the data.
+2. Ensure you've copied the URL or iFrame snippet available in dbt Explorer's **Data health** section, under the **Embed data health into your dashboard** toggle.
+3. Add a new embedded UI element in your Sigma Workbook in the following format:
+
+    ```html
+    https://metadata.ACCESS_URL/exposure-tile?uniqueId=exposure.EXPOSURE_NAME&environmentType=production&environmentId=ENV_ID_NUMBER&token=<YOUR_METADATA_TOKEN>
+    ```
+
+    *Note, replace the placeholders with your actual values.*
+4. You should now see the data health tile embedded in your Sigma dashboard.
+
+</TabItem>
+
```

Comment:
Hiya @mirnawong1 

I'm not sure if this would be OTT as there are a few pics already but would it be worth adding a pic showing what the data health tile in the Sigma dashboard would look like?

Kind Regards
Natalie

Reply:
i think that's what this is?  with the bonus of how to embed the URL:

![Screenshot 2024-10-22 at 11 00 09](https://github.com/user-attachments/assets/5747a3db-a8d0-4de3-9c78-33a7652836cf)